### DISCUSSION TITLE
[evol] On/Off button

### DISCUSSION THREAD

**Discussion by adyb0** [MathRobin/Never-Consent#67] [Ideas]
hello,

Some website functionnality sometime need a consent.
I see some website using recaptcha on the loggin page, with an RGPD consent needed to do it but not classified as "nessesary" (because unlogged content dont need?? i think). 

Do you think that could be a good idea to add this when we click on "never-consent" icon on the toolbar ?

Regards

**Comment by MathRobin** [comment]
yes why not, i have the reflex to disable the extension when i need but yes, an on/off button could be a great idea.

I'm thinking a v2 of this extension, will be inside

**Comment by MathRobin** [comment]
Have created an issue (https://github.com/MathRobin/Never-Consent/issues/68) and have added it to V2 project (https://github.com/MathRobin/Never-Consent/projects/1)
